android/src/main/java/cloud/pace/plugins/cloudsdk/CloudSDK.kt

```diff
@@ -234,22 +287,22 @@ class CloudSDK : Plugin(), AppCallback {
     override fun onImageDataReceived(bitmap: Bitmap) {
     }
 
+    override fun onLogin(context: Context, result: Completion<String?>) {
+    }
+
+    override fun onLogout(onResult: (LogoutResponse) -> Unit) {
```

Comment:
Do we need all these methods?

Reply:
As discussed: This is currently still needed, because the default implementation is not applied. Will be investigated in a follow-up.